app/Repositories/PostRepository.php

```diff
@@ -85,7 +85,8 @@ public function create(array $data, $signupId)
             'status' => isset($data['status']) ? $data['status'] : 'pending',
             'source' => isset($data['source']) ? $data['source'] : null,
             'remote_addr' => isset($data['remote_addr']) ? $data['remote_addr'] : null,
-
+            'type' => isset($data['type']) ? $data['type'] : 'photo',
+            'action' => isset($data['action']) ? $data['action'] : 'default',
         ]);
```

Comment:
are these the defaults we want to include if `type` and `action` are not sent in the request?

Reply:
yea